fix(typos): repeated words (#20848)
mdn/content: files/en-us/glossary/call_stack/index.md

- When a script calls a function, the interpreter adds it to the call stack and then starts carrying out the function.
- Any functions that are called by that function are added to the call stack further up, and run where their calls are reached.
- When the current function is finished, the interpreter takes it off the stack and resumes execution where it left off in the last code listing.

> - If the stack takes up more space than it was assigned, then this results in a "stack overflow" error.

Reviewer: ```suggestion
- If the stack takes up more space than it was assigned, a "stack overflow" error is thrown.
```
